### DISCUSSION TITLE
Thumbnail captions

### DISCUSSION THREAD

**Discussion by anjapetry** [dailydotdev/daily#927] [Feature Requests]
Hi, daily.dev team! 

I recently shared a link to a YouTube video by Addy Osmany on 'The Cost Of JavaScript - 2023': https://app.daily.dev/posts/CcXhhg1Fn#c-ix9Guzns3.

Several readers commented that they would have appreciated to know beforehand that they were being directed to a video instead of an article.

Therefore, I'd like to suggest adding thumbnail captions that state what type of content is being shared: an article, a podcast or a video. A thumbnail caption may help busy developers decide whether or not to spend time on content.

Kind regards, Anja

**Comment by idoshamun** [comment]
Hi @anjapetry,

Supporting videos and specifically YouTube is on our roadmap.
Once we officially support it we would add an indication that a post is actually a video.
Podcasts unfortunately are not yet on the roadmap.
I hope this makes sense

**Reply by anjapetry** [reply]
Hi @idoshamun,

Yes, this makes perfect sense, and I'm sure the community will love more curated video content. Greatly appreciate adding a content type indicator. Thanks for the update
